src/components/dashboard/main-content/course-enrollments/course-cards/UpgradeCourseButton.jsx

```diff
@@ -33,25 +84,32 @@ const UpgradeCourseButton = ({
       'edx.ui.enterprise.learner_portal.course.upgrade_button.clicked',
     );
   };
-
   const handleEnroll = () => {
     sendEnterpriseTrackEvent(
       enterpriseCustomer.uuid,
       'edx.ui.enterprise.learner_portal.course.upgrade_button.to_ecommerce_basket.clicked',
     );
   };
 
+  const upgradeButtonScreenReaderText = useCallback((chunks) => {
+    (<span className="sr-only">{chunks}</span>);
+  }, []);
```

Comment:
It looks like the `span` here isn't actually getting rendered in the DOM:

<img width="874" alt="image" src="https://github.com/user-attachments/assets/ee624ce1-d08d-4dec-9a70-b94cbbec4405">